packages/core/database/lib/transaction-context.js

```diff
@@ -10,7 +10,8 @@ const transactionCtx = {
   },
 
   get() {
-    return storage.getStore();
+    const trx = storage.getStore();
+    return trx?.isCompleted() ? undefined : trx;
```

Comment:
Should we automatically remove the local stored value when transactions are done?